pynicotine/gtkgui/userbrowse.py

```diff
@@ -358,8 +366,25 @@ def on_expand(self, widget):
             self.expand.set_property("icon-name", "go-down-symbolic")
 
     def on_folder_row_activated(self, treeview, path, column):
-        if self.user != config.sections["server"]["login"]:
-            self.on_download_directory()
```

Comment:
It is way too easy to download a folder, as there is no confirmation, this leads to accidental downloads (evidenced by transfers that get immediately cancelled) so I have changed the folder row activation to Expand / Collapse instead.

In the future I intend to restore the ability to download a folder with mouse-click, by using a <Control> modifier keymask, which would use the same conditions as the "Return" key binding, so that code may be moved into here at some point, when I can figure out how to pass on_key_press_event_folder(*args) into here.